Makie/src/utilities/texture_atlas.jl

```diff
@@ -717,7 +717,7 @@ function get_glyph_sdf(atlas::TextureAtlas, hash::UInt32)
 end
 
 function glyph_boundingbox(::BezierPath, ::Makie.NativeFont)
-    # TODO, implement this
+    # TODO:, implement this
     # Main blocker is the JS side since this is a bit more complicated.
     return (Vec2f(0), Vec2f(0))
 end
```

Comment:
Iirc this was added for WGLMakie but wasn't needed? Can we remove this? Or mark this as a dummy method if it's needed as one?

Reply:
No this is needed.
It's called here:
![image](https://github.com/user-attachments/assets/6ea9893b-826b-4a98-8889-aaa99415ae6f)
And the result won't be used - but should be as soon as we have time to implement it.